### PR TITLE
Tweak back button

### LINKED ISSUE #421: Mentor Discussion: Back Button tweaks
https://github.com/exercism/v3-website/blob/c4fe5f0bc74ca213d094b5798c3523fc858ae649/app/javascript/components/mentoring/Discussion.tsx#L73

- This is probably better named `CloseButton` as it's a cross in the newer designs.
- It should link to mentor dashboard, not exercise (for mentors)

### PR DESCRIPTION
**Pull request by kntsoriano**
Closes https://github.com/exercism/v3-website/issues/421.

### CODE REVIEW

File: app/javascript/components/mentoring/discussion/CloseButton.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { GraphicalIcon } from '../../common'
+
+export const CloseButton = ({ url }: { url: string }): JSX.Element => (
+  <a href={url} className="close-btn" aria-label="Close discussion">
```
Comment (SleeplessByte) [inline app/javascript/components/mentoring/discussion/CloseButton.tsx]: This is fine. It's friendlier to actually tell the screenreader _where_ the user is going to. `Close discussion and go to X`

**Review by SleeplessByte — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Comment by iHiD** [comment]
Needs a rebase

File: app/javascript/components/mentoring/discussion/CloseButton.tsx (hunk 1 above)
Comment (iHiD) [reply app/javascript/components/mentoring/discussion/CloseButton.tsx]: This got fixed in another branch.

**Comment by iHiD** [comment]
I feel like this has got done twice. Once in this branch and once in another :)

**Comment by kntsoriano** [comment]
@iHiD Okay. I will merge those changes.

**Pull request merged**